pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -210,7 +210,7 @@ class MindtPySolver(object):
         domain=bool
     ))
     CONFIG.declare("add_integer_cuts", ConfigValue(
-        default=True,
+        default=False,
```

Comment:
I'm still debating what is best to do here. In general, the integer cuts are not required for convex programs, and we do not have an implementation for general Integer variables yet (only binaries so far). Including them might hurt our capability of providing valid lower bounds to the optimal solution.
DICOPT has them on by default, Pajarito doesn't.